core/http/client/src/main/java/org/eclipse/rdf4j/http/client/SPARQLProtocolSession.java

```diff
@@ -102,6 +86,26 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.StringReader;
+import java.net.HttpURLConnection;
+import java.net.URISyntaxException;
+import java.nio.charset.Charset;
+import java.nio.charset.IllegalCharsetNameException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ExecutorService;
+
+import static org.eclipse.rdf4j.http.protocol.Protocol.ACCEPT_PARAM_NAME;
```

Comment:
Don't bother fixing it for this PR, but: could you fix your formatter to not reorganize imports in this fashion? Most of our code follows the Eclipse convention to have static imports first, then normal imports organized with anything from java.* at the top, then javax.*, then org.*, then com.*. 

Reason I ask is that I've had to resolve some merge conflicts in the past that were caused by imports being moved around, and it's a hassle.

This was never made explicit because in the past everyone on the project used Eclipse, so it was just the implicit default. Perhaps we should document this now that we have a more diverse toolset.

Reply:
Doesn’t everyone almost everyone use IntelliJ now? Since this is the default with IntelliJ. Btw, I’m not a fan of having to adjust my editor for every project, there was once an attempt at making a standardized editor config, maybe we should look at that. Also I think the maven formatter plugin should handle this and if it doesn’t then we need to just live with the fact that the imports are going to change. I’ll try not to optimize imports for files I don’t touch. And also IntelliJ can fix import conflicts for you very easily.

Reply:
> Doesn’t everyone almost everyone use IntelliJ now?

No.

> Since this is the default with IntelliJ. Btw, I’m not a fan of having to adjust my editor for every project, there was once an attempt at making a standardized editor config, maybe we should look at that.

To some extent this is unavoidable in an open-source world I'm afraid. Every project has different conventions. 

Then again this one is not a huge deal. I'll cop the occasional merge conflict fallout (it doesn't happen very often anyway - mostly it can resolve it fine, and even if it doesn't it's easy to fix in the IDE).

> Also I think the maven formatter plugin should handle this and if it doesn’t then we need to just live with the fact that the imports are going to change. I’ll try not to optimize imports for files I don’t touch.

I'll take a look at the formatter plugin, see if that's possible. Like I said earlier, we haven't really enforced this guideline before, and I'm not even sure what Netbeans (which Bart uses) does, so don't worry too much about it - I merely asked on the off chance it was a simple tweak on your end.